src/commands/init.ts

```diff
@@ -3,19 +3,19 @@
  */
 
 import { Command, flags } from '@oclif/command';
+import { OutputFlags } from '@oclif/parser';
 import cli from 'cli-ux';
 import * as execa from 'execa';
 import * as fs from 'fs';
 import * as path from 'path';
+import { testFiles } from '../templates/init/test/test-file-info';
+import { CopyDescriptor } from '../utils/copy-list';
+import { InitializationType, packageJsonParts } from '../utils/initialization-helper';
 import { copyFiles, ensureDirectoryExistence, findConflicts, readTemplates } from '../utils/templates';
 
-const backendScripts = {
+const backendBuildScripts = {
   'ci-build': 'echo "Use this to compile or minify your application"',
-  'ci-package': 'echo "Copy all deployment-relevant files to the `deployment` folder"',
-  'ci-backend-unit-test':
-    'echo "Test your application and write results in a JUnit format to `s4hana_pipeline/reports/backend-unit/` and coverage in a cobertura format to `s4hana_pipeline/reports/coverage/backend-unit/`"',
-  'ci-integration-test':
-    'echo "Test your application and write results in a JUnit format to `s4hana_pipeline/reports/backend-integration/` and coverage in a cobertura format to `s4hana_pipeline/reports/coverage/backend-integration/`"'
+  'ci-package': 'echo "Copy all deployment-relevant files to the `deployment` folder"'
```

Comment:
Maybe all scripts should be moved to the initHelper.ts

Reply:
move it too the helper